deploy/helm/sumologic/templates/otelcol-deployment.yaml

```diff
@@ -0,0 +1,70 @@
+{{- if eq .Values.sumologic.traces.enable true }}
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ printf "%s-otelcol" (include "sumologic.name" .) }}
+  labels:
+    app: {{ printf "%s-otelcol" (include "sumologic.labels.app" .) }}
```

Comment:
any reason not to move this logic to the tpl so you can just call template?

Reply:
I just wanted to be consistent with events deployment:  https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/master/deploy/helm/sumologic/templates/events-deployment.yaml#L5

Should we change that in both places?